ddl/ddl_api.go

```diff
@@ -568,14 +569,20 @@ func checkColumnValueConstraint(col *table.Column) error {
 	return nil
 }
 
-func checkDuplicateColumn(colDefs []*ast.ColumnDef) error {
-	colNames := map[string]bool{}
-	for _, colDef := range colDefs {
-		nameLower := colDef.Name.Name.L
-		if colNames[nameLower] {
-			return infoschema.ErrColumnExists.GenWithStackByArgs(colDef.Name.Name)
+func checkDuplicateColumn(cols []interface{}) error {
```

Comment:
How about create a new function `checkDuplicateColumnName(cols []model.CIStr)` to do this？

Reply:
Same check logical with different input parameters, so I think rewrite existing `checkDuplicateColumnNameis` better.